windows/src/desktop/kmshell/locale/zh-CN/strings.xml

```diff
@@ -19,7 +19,7 @@
   <!-- Context: Formatted Messages -->
   <!-- String Type: FormatString -->
   <!-- Introduced: 7.0.230.0 -->
-  <string name="SKButtonYes" comment="Yes button in message boxes">&amp;是</string>
+  <string name="SKButtonYes" comment="Yes button in message boxes">是 (&amp;Y)</string>
```

Comment:
There will be many other strings like this (anywhere we see `&amp;` I suspect)

Reply:
The latest commit https://github.com/keymanapp/keyman/pull/5681/commits/6bf1ef004ac427ef5eaa5ded6fbdba7ecbc63577 reverts to the English hotkeys